code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -987,3 +1031,6 @@
 	statpanel_name = "Metal"
 	stack = /obj/item/stack/sheet/metal
 	add_to_storage = TRUE
+
+
+
```

Comment:
```suggestion

```
not totally sure if this is showing up correctly, unneeded whitespace on 1035, and 1036